pkg/quota/admission/runonceduration/admission.go

```diff
@@ -30,14 +30,14 @@ func init() {
 }
 
 func readConfig(reader io.Reader) (*api.RunOnceDurationConfig, error) {
```

Comment:
This method is getting repeated a lot.  It looks like the intent is `ReadYAML` as opposed to the existing `ReadYamlInto` behavior (yeah, poorly named).

How about doing that refactor.